pkg/events/events.go

```diff
@@ -5047,6 +5047,7 @@ var Definitions = eventDefinitions{
 				TailCalls: []TailCall{
 					{MapName: "prog_array_tp", MapIndexes: []uint32{tailSchedProcessExecEventSubmit}, ProgName: "sched_process_exec_event_submit_tail"},
 				},
+				Capabilities: []cap.Value{cap.SYS_PTRACE}, // CopyRegularFileByRelativePath()
```

Comment:
sched_process_exec by itself shouldn't raise this capability.
It should be set only when capture exec (which has its own event) is set or hash calculation is required.